src/libs/antares/study/links/property-links-helper.h

```diff
@@ -7,13 +7,13 @@ namespace Antares
 {
 namespace Data
 {
-std::string transmissionCapacitiesToString(const TransmissionCapacities& tc)
+std::string transmissionCapacitiesToString(const LocalTransmissionCapacities& tc)
```

Comment:
I'm not a fan of this source **.h** file.
Indeed, for other parameters, the **enum** and the functions to act on the value of type **enum** are in **fwd.h/cpp**. Here, functions are defined in **property-links-helper.h**.

A data, some functions to interact with the data. It's a class.
I guess every parameters should be encapsulated in a class defined in a separate source file.